apps/shooting-clubs/translations/src/en/pages.json

```diff
@@ -10,7 +10,7 @@
   },
   "second-contact-name": {
     "header": "Who should we contact in {{values.club-secretary-name}}'s absence?",
```

Comment:
I would need to see precisely what Vijay and you are entering. It could be that Vijay used a back tick or pasted the name into the field from a formatted text. Anyway, I'll merge this and Vijay can watch out for other problems